src/xcode/ENA/ENA/Source/Scenes/Home/Cells/TestResult/HomeTestResultCellModel.swift

```diff
@@ -30,16 +59,20 @@ class HomeTestResultCellModel {
 					}
 
 					self?.configure(for: pcrTest.testResult)
-					onUpdate()
+					self?.onUpdate()
 				}
 				.store(in: &subscriptions)
 
 			coronaTestService.$pcrTestResultIsLoading
 				.receive(on: DispatchQueue.OCombine(.main))
 				.sink { [weak self] testResultIsLoading in
-					if testResultIsLoading && self?.coronaTestService.pcrTest?.finalTestResultReceivedDate == nil {
-						self?.configureLoading()
-						onUpdate()
+					if self?.coronaTestService.pcrTest?.finalTestResultReceivedDate == nil {
+						if testResultIsLoading {
```

Comment:
isTestResultLoading ?

Reply:
This way it makes a correct english sentence: "If test result is loading [do x]" I admit being inconsistent with this, but would like to keep it that way.

Reply:
It's more a personal style and how you prefer, actually I saw isActivityIndicatorHidden in the next line, so in that context, I found it to be little inconsistent.